lib/repo/subversion_repository.rb

```diff
@@ -35,19 +35,23 @@ class SubversionRepository < Repository::AbstractRepository
     # created using SubversionRepository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
-        raise ConfigurationError.new("Required config 'IS_REPOSITORY_ADMIN' not set")
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise ConfigurationError.new("Init: Required config " \
+                                     "'IS_REPOSITORY_ADMIN' not set")
       end
-      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
-        raise ConfigurationError.new("Required config 'REPOSITORY_PERMISSION_FILE' not set")
+      if MarkusConfigurator.markus_config_repository_permission_file.nil?
+        raise ConfigurationError.new("Required config " \
+                                     "'REPOSITORY_PERMISSION_FILE' not set")
       end
       begin
         super(connect_string) # dummy call to super
       rescue NotImplementedError; end
       @repos_path = connect_string
       @closed = false
-      @repos_auth_file = Repository.conf[:REPOSITORY_PERMISSION_FILE] || File.dirname(connect_string) + "/svn_authz"
-      @repos_admin = Repository.conf[:IS_REPOSITORY_ADMIN]
+      @repos_auth_file = MarkusConfigurator.
+                           markus_config_repository_permission_file 
+                         || File.dirname(connect_string) + '/svn_authz'
```

Comment:
@reidka I think the line break is confusing here. Maybe put the whole expression in parens:

```
@repos_auth_file = (MarkusConfigurator.
                            markus_config_repository_permission_file 
                          || File.dirname(connect_string) + '/svn_authz')
```